kop/java/pom.xml

```diff
@@ -41,6 +41,12 @@
       <version>${slf4j.simple.version}</version>
       <scope>runtime</scope>
     </dependency>
+
+    <dependency>
+      <groupId>io.streamnative.pulsar.handlers</groupId>
+      <artifactId>oauth-client</artifactId>
+      <version>2.9.2.8</version>
```

Comment:
> Failure to find io.streamnative.pulsar.handlers:oauth-client:jar:2.9.2.8 in https://repo.maven.apache.org/maven2 was cached in the local repository

The release CI for `oauth-client` has been broken for a long time. See https://github.com/streamnative/streamnative-ci/pull/1657, the first release available will be 2.9.2.13.

Reply:
Oh, the looks like 2.9.2.8 come from my local cache, we need to await the new version release to the maven center.